tests/WE2E/test_configs/grids_extrn_mdls_suites_community/config.grid_RRFS_CONUScompact_25km_ics_FV3GFS_lbcs_RAP_suite_RAP_quilt_off.yaml

```diff
@@ -0,0 +1,33 @@
+metadata:
```

Comment:
I think this test would fit better under the "wflow_features" directory, maybe a name like "config.quilting_off.yaml". If you'd like to keep it here I'd suggest a symlink in that directory, since this is a major feature that's being tested.

Reply:
@mkavulich Moved and renamed test to `quilting_false`.